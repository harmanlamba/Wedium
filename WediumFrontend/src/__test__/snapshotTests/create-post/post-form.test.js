import React from 'react';
import renderer from 'react-test-renderer';
import {
  shallow
} from 'enzyme';

import PostForm from '../../../components/create-post/post-form';

it('snapshot postform default load', () => {
  const setRouteLeaveHook = jest.fn();

  let wrapper = shallow( <
    PostForm.WrappedComponent params = {
      {
        router: setRouteLeaveHook,
      }
    }
    />
  );

  expect(wrapper).toMatchSnapshot();
});