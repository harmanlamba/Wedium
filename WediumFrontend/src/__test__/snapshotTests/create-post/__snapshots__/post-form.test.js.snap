// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot postform default load 1`] = `
<PostForm
  classes={
    Object {
      "container": "PostForm-container-1",
      "textField": "PostForm-textField-2",
    }
  }
  params={
    Object {
      "router": [MockFunction],
    }
  }
/>
`;
